templates/web/base/report/updates.html

```diff
@@ -6,6 +6,9 @@
   [%- END %]
   [%- NEXT %]
 [%- END %]
+[%- IF update.get_extra_metadata('triage_report') == 1 AND ( NOT c.user OR ( NOT c.user.from_body AND NOT c.user.is_superuser ) ) %]
```

Comment:
Feels like a 'staff' state for updates might be better here, but something for later perhaps.